src/trigger/scripttrigger.cpp

```diff
@@ -55,7 +58,9 @@ ScriptTrigger::ScriptTrigger(const Vector& pos, const std::string& script_) :
   triggerevent(EVENT_TOUCH),
   script(script_),
   new_size(),
-  must_activate()
+  must_activate(),
+  oneshot(false),
+  runcount(0)
```

Comment:
I am not sure if initializing these here to those values is the right thing to do. @tobbi, what can you say about this?

Reply:
Yes, this is th right thing to do. Why wouldn't it?

Reply:
Because, compared to the first constructor, where there is `must_activate(false)`, `must_activate()` is used here. What does leaving out the argument do?

Reply:
Use the default value for the data type, which, coincidentally happens to be ```false```.